README.md

```diff
@@ -145,7 +113,7 @@ $ ./rendezvous
 Open a new terminal window and start the SDO IoT Platform SDK services:
 ```
 $ cd <sdo-iot-platform-sdk-root>/demo
-$ sudo docker-compose up
+$ docker-compose up
```

Comment:
docker-compose up --build ?

Reply:
Haven't been using the build flag in the previous releases. Added it now.